invenio_app_rdm/config.py

```diff
@@ -429,6 +429,10 @@ def files_rest_permission_factory(obj, action):
         "task": "invenio_accounts.tasks.delete_ips",
         "schedule": timedelta(hours=6),
     },
+    "update_domain_status": {
+        "task": "invenio_accounts.tasks.update_domain_status",
+        "schedule": timedelta(minutes=60),
```

Comment:
```suggestion
        "schedule": timedelta(hours=4),
```

minor: a bit arbitrary, but since the underlying query is also performing updates to the database, I think seeing updates 2-3 times during working hours is good enough.

Reply:
Note, the task doesn't update the index, just the db. Probably the task https://github.com/inveniosoftware/invenio-accounts/blob/24f9ac29879f2daa7e49c94c0370444fc4dc89aa/invenio_accounts/tasks.py#L81 should be updated to also index the domains that it changes.

Reply:
Once we've updated the task to also reindex (using the bulk queue), I would say we merge and see if it needs adjustments in terms of the regularity, bulk size, or similar.